lib/scsi_defs.h

```diff
@@ -2,9 +2,13 @@
 #define LIB_SCSI_DEFS_H
 
 #include <cstdint>
+#include "absl/base/attributes.h"
+
 
 namespace scsi_defs {
 
+  constexpr uint32_t kInquiryCommandSize = 6;
```

Comment:
no longer doing these constants https://github.com/googleinterns/scsi2nvme/pull/29#discussion_r428935909

Reply:
I thought it was only for the read, write, sync and verify structs because of their naming. Let me link this discussion on #29 and have folks pitch in

Reply:
Responded there, but tl;dr they don't seem necessary.